.github/ISSUE_TEMPLATE/creating-a-new-release.md

```diff
@@ -56,6 +56,9 @@ git push --delete upstream <tag_name>
 git clean -x -d -n
 # before running the next command check, that it is ok to remove the files
 git clean -x -d -f
+# Please remove python3-setuptools_scm, or it will all git files into tarbal.
```

Comment:
I guess it would be safer to do create a temp-directory, extract the result from `git archive` there and continue with the other steps there to ensure it is a clean cope. Seems safer that using git clean.

Reply:
That complex things. I prefer git clean way which does not require creating new folder.